src/appdistribution/client.ts

```diff
@@ -72,17 +72,63 @@ export interface Group {
   inviteLinkCount?: number;
 }
 
+export interface CreateReleaseTestRequest {
+  releaseTest: ReleaseTest;
+}
+
+export enum TestState {
```

Comment:
Optional: Consider a string union type here instead (ie `IN_PROGRESS" | "PASSED" | "FAILED" | "INCONCLUSIVE`). They tend to be a bit easier to work with IMO.

Reply:
Thanks for the tip! Done.